pkg/server/commands/list_objects.go

```diff
@@ -114,6 +119,8 @@ func (q *ListObjectsQuery) handler(
 		// ConnectedObjects currently only supports models that do not include intersection and exclusion,
 		// and the model must include type info for ConnectedObjects to work.
 		if !containsIntersection && !containsExclusion && hasTypeInfo {
+			span.SetAttributes(attribute.Bool("list-objects-optimized", true))
```

Comment:
the previous `set` is inside `handler`... should we do the same here and put it in line 150?

Reply:
Yeah, good call 👍

Reply:
Ok, moved.